test/fixtures/generatedSwagger.json

```diff
@@ -1,719 +1,2509 @@
 {
```

Comment:
tried the generatedSwagger.json fixture on http://editor.swagger.io/ and got this
 
![Screenshot 2020-05-30 at 17 53 05](https://user-images.githubusercontent.com/10440327/83339693-6d451d80-a29e-11ea-974f-714fcc5a1263.png)

Reply:
because of the shortcut blueprints. line 628
```yaml
/pet/create:
    get:
      summary: Create Pet *
      description: |-
        Create a new **Pet** record.

        (\*) Note that this is a [Sails blueprint shortcut route](https://sailsjs.com/documentation/concepts/blueprints/blueprint-routes#?shortcut-blueprint-routes) (recommended for **development-mode only**)
      externalDocs:
        url: 'https://sailsjs.com/documentation/reference/blueprint-api/create'
        description: 'See https://sailsjs.com/documentation/reference/blueprint-api/create'
      tags:
        - Pet
      parameters: []
      responses:
        '200':
          description: Responds with a JSON dictionary representing the newly created **Pet** instance
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/pet'
        '404':
          description: Resource not found
        '500':
          description: Internal server error
      requestBody:
        description: JSON dictionary representing the Pet instance to create.
        required: true
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/pet'
```

the `schema` should have been under `parameters` as `query` param https://sailsjs.com/documentation/concepts/blueprints/blueprint-routes#?shortcut-blueprint-routes

Reply:
I uploaded to http://editor.swagger.io/ but without shortcuts. Fixing now.

Reply:
Done; see new commits

Reply:
just checked, all good 👍 , but noticed the `generateSchemaAsQueryParameters` didn't work for one of the shortcut blueprint action 
![Screenshot 2020-06-01 at 20 19 47](https://user-images.githubusercontent.com/10440327/83466513-4b39cf80-a445-11ea-87a1-2bcabbace90d.png)

Reply:
Thanks. Looks to be a side-effect of overriding `user/create` action https://github.com/theoomoregbee/sails-hook-swagger-generator/blob/refactor-parsing/api/controllers/UserController.js#L76

Will look.